### ISSUE TITLE
Giveth Ethereum blockchain node

### ISSUE THREAD

**Issue by vsimetka** [Giveth/milestonetracker-ui#12]
Set up, run and integrate our own blockchain node so that the enduser can use our DAPP without need to run Geth or Metamask.

**Comment by GriffGreen** [comment]
Yes please! this is needed to bring transparency to our project and every project that is associated with giveth, so that any one can see how the funds are being used.

**Comment by jbaylina** [comment]
The solution to this is that the DAPP detects an ethereum client or not. In case of not detection it should go to the server and alert the user. I'll set the server and modify the code for the detection.
